src/main/java/ch/naviqore/raptor/router/LabelPostprocessor.java

```diff
@@ -375,7 +375,8 @@ private boolean canStopTimeBeTarget(StopTime stopTime, int routeSourceTime, Time
             return null;
         }
 
-        return stopTimes[firstStopTimeIdx + tripOffset * numberOfStops + stopOffset];
+        int stopTimeIndex = firstStopTimeIdx + 2 * (tripOffset * numberOfStops + stopOffset) + 2;
```

Comment:
I played around a bit, but think it's not beneficial for following reasons:

- Almost all StopTime accessors are very unique and there is no repeat except for once and there are relative complex methods that require a lot of datastructures from the methods using the stoptimes array (especially during creation), thus there is a lot of value copying needed to copy over all arguments to the StopTimeUtils method. Which I think would impact readability. Also I personally think it makes it more difficult to read because the utils method would be very specific to what the calling method.